samples/bluetooth/unicast_audio_server/src/main.c

```diff
@@ -69,11 +69,24 @@ static const struct bt_data ad[] = {
 	BT_DATA(BT_DATA_SVC_DATA16, unicast_server_addata, ARRAY_SIZE(unicast_server_addata)),
 };
 
+#define AUDIO_DATA_TIMEOUT_US 1000000UL /* Send data every 1 second */
+#define SDU_INTERVAL_US       10000UL   /* 10 ms SDU interval */
+
 static uint16_t get_and_incr_seq_num(const struct bt_bap_stream *stream)
 {
+	uint16_t seq_num;
+
 	for (size_t i = 0U; i < configured_source_stream_count; i++) {
 		if (stream == &source_streams[i].stream) {
-			return source_streams[i].seq_num++;
+			seq_num = source_streams[i].seq_num;
+
+#if defined(CONFIG_LIBLC3)
+			source_streams[i].seq_num++;
+#else
+			source_streams[i].seq_num += (AUDIO_DATA_TIMEOUT_US /
+						      SDU_INTERVAL_US);
+#endif
```

Comment:
```suggestion
			if (IS_ENABLED(CONFIG_LIBLC3)) {
				sinks[i].seq_num++;
			} else {
				sinks[i].seq_num += (AUDIO_DATA_TIMEOUT_US /
						     codec_configuration.qos.interval);	
			}
```

Reply:
Alternatively you can also do
```suggestion
			if (IS_ENABLED(CONFIG_LIBLC3)) {
				return sinks[i].seq_num++;
			} else {
				return sinks[i].seq_num += (AUDIO_DATA_TIMEOUT_US /
							    codec_configuration.qos.interval);	
			}
```

And remove the local variable again

Reply:
Updated.